Expansion of Examples for deviceOwnership property

alexlushsevernunival opened the pull request:
Allowed values column should include the allowed values which in this case I think is just the three strings. Usage should include one of the known string values rather than -eq "".

azurecla commented:
Hi __@felixrieseberg__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (feriese). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

jdconley commented on articles/nodejs-use-node-modules-azure-apps.md:
```diff
@@ -29,11 +29,14 @@ When deploying the **node\_modules** directory as part of your application, it w
-While most modules are simply plain-text JavaScript files, some modules are platform-specific binary images. These modules are compiled at install time, usually by using Python and node-gyp. One specific limitation of Azure Webites is that while it natively understands how to install modules specified in a **package.json** or **npm-shrinkwrap.json** file, it does not provide Python or node-gyp an ...
+While most modules are simply plain-text JavaScript files, some modules are platform-specific binary images. These modules are compiled at install time, usually by using Python and node-gyp. One specific limitation of Azure Websites is that while it natively understands how to install modules specified in a **package.json** or **npm-shrinkwrap.json** file, it does not provide Python or node-gyp a ...
 
 Since Azure Cloud Services rely on the **node\_modules** folder being deployed as part of the application, any native module included as part of the installed modules should work in a cloud service as long as it was installed and compiled on a Windows development system. 
```
It's a bit confusing that this "Native Modules" section interleaves the concepts of Cloud Services and Websites. Maybe a subsection makes sense, since it is a bit different, and now most of this content is about Websites. On Cloud Services you have to ship native_modules (or write your own npm install script), but on Websites you only have to ship them if they don't build.

jdconley commented:
:+1: Copy flows much better now.

thauburger commented:
+1. Great update, @felixrieseberg!

alexlushsevernunival committed:
Expansion of Examples for deviceOwnership property

Allowed values column should include the allowed values which in this case I think is just the three strings. Usage should include one of the known string values.
